InfiniteLoopers-App/src/main/java/com/isa/account/LoginManager.java

```diff
@@ -1,30 +1,71 @@
 package com.isa.account;
 
+import java.util.List;
+
+import static com.isa.menu.Menu.menuInvoke;
+
 public class LoginManager {
-    private AuthenticationService authService;
-    private UserInputReader userInputReader;
+    public LoggedUser getLoggedUser() {
+        return loggedUser;
+    }
+
+    private LoggedUser loggedUser = new LoggedUser();
+    private static AuthenticationService authService;
+    private static UserInputReader userInputReader;
```

Comment:
to średnio może być statyczne jak na tym w konstruktorze jest robione `this`, może to być po prostu prywatne finalne pole, w sumie oba i logged user również